src/Microsoft.OData.Client/DataServiceContext.cs

```diff
@@ -50,7 +50,7 @@ internal enum UndeclaredPropertyBehavior
     /// The <see cref="T:Microsoft.OData.Client.DataServiceContext" /> represents the runtime context of the data service.
     /// </summary>
     [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Maintainability", "CA1506", Justification = "Central class of the API, likely to have many cross-references")]
-    public class DataServiceContext
+    public class DataServiceContext : IDataServiceContext

```

Comment:
>DataServiceContext : IDataServiceContext [](start = 17, length = 40)

So, how to test the "Mock" in your scenario?

Please add the Unit test, you can use "Moq" library in the unit test.

Reply:
Ok will add